locals.tf

```diff
@@ -10,10 +10,10 @@ locals {
 
   iam_activity = merge(
     {
-      Root = "{ $.userIdentity.type = \"Root\" }"
+      Root = "{$.userIdentity.type=\"Root\" && $.userIdentity.invokedBy NOT EXISTS && $.eventType != \"AwsServiceEvent\"}"
```

Comment:
Please leave the spaces in there...

Reply:
I left the spaces out because I wanted to be sure the filter was equal to the remediation option provided by AWS. Do you think it will auto-resolve if I put them back in?

Reply:
The spaces shouldn't matter for the filter logic.

Reply:
I'm well aware, but it's not about filter logic. This whole change is to improve security posture by getting green checks in security hub. Those checks will only pass when the filter matches _exactly_ to what is documented.

Reply:
I would hope they do not process the exact string as is, but that they parse it to a filter first. In which case it should be fine to leave the spaces and to keep our formatting consistent. If I'm wrong we can always add the spaces back later right?

Reply:
haha you'd hope indeed but things at AWS aren't always logical. My guess here is that the security hub check only matches the filter pattern string to what it requires, same way you'd get it when retrieving the metric filters through an API call. No parsing done at all.

But sure, we can try and fix it if it doesn't work. Will take some time to figure out as security hub doesn't check that often. Let me make the change.

Reply:
Made the change back to spaces.